src/VisualStudio/Core/Def/Implementation/FindReferences/Contexts/AbstractTableDataSourceFindUsagesContext.cs

```diff
@@ -368,7 +370,19 @@ private async Task<(ExcerptResult, SourceText)> ExcerptAsync(SourceText sourceTe
                 return (excerptResult, AbstractDocumentSpanEntry.GetLineContainingPosition(sourceText, documentSpan.SourceSpan.Start));
             }
 
-            private ImmutableDictionary<string, string> GetAggregatedCustomColumnsData(ImmutableDictionary<string, ImmutableArray<string>> customColumnsDataOpt)
+            private ImmutableDictionary<string, string> GetAggregatedCustomColumnsData(IEnumerable<KeyValuePair<string, ImmutableArray<string>>> customColumnsDataOpt, ImmutableArray<CodeAnalysis.AdditionalProperty> customColumns)
```

Comment:
Both parameter names look identical. Can this be more distinct names `customColumnsWithMultipleValuesOpt` and `customColumns`? Also considering using a single `ImmutableDictionary.Builder` to avoid allocating multiple dictionaries in this method.